olive/passes/pass_config.py

```diff
@@ -61,7 +59,7 @@ def get_user_script_config(
 ) -> Dict[str, PassConfigParam]:
     type_ = str
     if allow_path:
-        type_ = Union[Path, str]
+        type_ = OLIVE_RESOURCE_ANNOTATIONS
```

Comment:
>OLIVE_RESOURCE_ANNOTATIONS [](http://example.com/codeflow?start=16&length=26)

revert to Union[Path, str]?

Reply:
Emm, it is still the resouce_path. But we filter the case of aml resouce out for script_dir and user_scriptl

Reply:
where do we filter them? I suggest to revert it to Union[Path, str] to keep consistence since all other places about script_dir and user_script is just local path.

the same with aml_system.py(model_script, script_dir).

Reply:
Updated to revert back to path|str for all user_script and script_dir